js/buttons.js

```diff
@@ -51,8 +51,8 @@ function addButtons() {
   // button category use values
   buttons.append('text')
     .classed('category-amount', true)
-    .text(function(d){
-      return 'XX';
+    .attr('id', function(d){
+      return (d + "_button_text");
```

Comment:
let's use kebab-case instead of snake_case per https://github.com/USGS-VIZLAB/vizlab/wiki/Style-guide bullet 6